packages/shared/src/mock/metadata-utils.ts

```diff
@@ -303,6 +303,7 @@ export function makeNxDataGroup<
       ...(auxAttr ? [makeNxAuxAttr(auxAttr)] : []),
     ],
     children: [
+      ...(groupOpts.children ?? []),
```

Comment:
Small modification to be able to add the `secondary_errors` dataset below

Reply:
I think you should add `children` to the destructuring statement instead (with `[]` as default value), so it doesn't get included in `groupOpts` in the first place.

Reply:
Sounds good, I will do the same for `attributes` above